doc/man/8/ceph-kvstore-tool.rst

```diff
@@ -9,14 +9,14 @@
 Synopsis
 ========
 
-| **ceph-kvstore-tool** <leveldb|rocksdb|bluestore-kv> <store path> *command* [args...]
+| **ceph-kvstore-tool** <rocksdb|bluestore-kv> <store path> *command* [args...]
 
 
 Description
 ===========
 
 :program:`ceph-kvstore-tool` is a kvstore manipulation tool. It allows users to manipulate
-leveldb/rocksdb's data (like OSD's omap) offline.
+rocksdb's data (like OSD's omap) offline.
```

Comment:
Please change s/rocksdb/RocksDB

Reply:
I have change all the rocksdb to RocksDB in the comment and docs. Please take a review.